docs/book/releases/v2.0.0.md

```diff
@@ -80,6 +80,14 @@
 10. `user` and `ca-file` change in the [vsphere-config-secret](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/installation.html#create-a-configuration-file-with-vsphere-credentials-) is not honored until vSphere CSI Controller Pod restarts.
     - Impact: Volume lifecycle operations fail until the controller pod restarts.
     - Workaround: Restart `vsphere-csi-controller` deployment Pod.
+11. vCenter restart may leave Kubernetes persistent volume claims in the Pending state with error message - `failed to get taskInfo for CreateVolume task from vCenter "vCenter IP" with err: ServerFaultCode: The session is not authenticated`.
```

Comment:
Actually, we found out recently that all pending CNS tasks are marked as `Failed` if CNS restarts. So even if you get by the not authenticated issue, it will still remain in Pending state for an hour. How about not mentioning the exact error message here?

Reply:
done. removed specific error message.